web/src/routes/MyPages/ProfilePage.tsx

```diff
@@ -268,6 +289,7 @@ export default function MyPage() {
                 onBlur={handleInputChange}
                 theme={theme}
                 disabled={!isDisabled}
+                maxLength={title === '전화번호' ? 13 : undefined}
```

Comment:
Same above. Do we need max 13 length?

Reply:
hmm... i think phoneNumber has to conflicted max length. if we release the limit phoneNumber Length, it has many problem than it conflict limit length. (i think...)

Reply:
No, I mean why not `12`?